draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -3978,35 +3847,61 @@ RESPONSE:
   Date: Wed, 04 May 2016 08:48:11 GMT
   Content-Type: application/cdni; ptype=ci-trigger-collection
 
-  {
-    "cdn-id": "AS64496:0",
-    "coll-status": [ 
-      { "status": "pending", "collection": "/triggers/pending" },
-      { "status": "active", "collection": "/triggers/active" },
-      { "status": "complete", "collection": "/triggers/complete" },
-      { "status": "processed", "collection": "/triggers/processed" },
-      { "status": "failed", "collection": "/triggers/failed" },
-      { "status": "cancelling",
-        "collection":"/triggers/cancelling" },
-      { "status": "cancelled", "collection": "/triggers/cancelled" }
-    ],
-    "coll-label": [ 
-      { "label": "1b1bad0c", "collection": "/triggers/1b1bad0c" },
-      { "label": "fafa9a97", "collection": "/triggers/fafa9a97" }
-    ],
-    "staleresourcetime": 86400,
-    triggers": [
-          "https://dcdn.example.com/triggers/0",
-          "https://dcdn.example.com/triggers/1"
-      ]
-  }
+{
+  "cdn-id": "AS64496:0",
+  "coll-status": [
+    {
+      "status"    : "pending",
+      "collection": "/triggers/pending"
+    },
+    {
+      "status"    : "active",
+      "collection": "/triggers/active"
+    },
+    {
+      "status"    : "complete",
+      "collection": "/triggers/complete"
+    },
+    {
+      "status"    : "processed",
+      "collection": "/triggers/processed"
+    },
+    {
+      "status"    : "failed",
+      "collection": "/triggers/failed"
+    },
+    {
+      "status"    : "cancelling",
+      "collection": "/triggers/cancelling"
+    },
+    {
+      "status"    : "cancelled",
+      "collection": "/triggers/cancelled"
+    }
+  ],
+  "coll-label": [
+    {
+      "label"     : "1b1bad0c", 
+      "collection": "/triggers/1b1bad0c"
+    },
+    {
+      "label"     : "fafa9a97",
+      "collection": "/triggers/fafa9a97"
+    }
+  ],
+  "staleresourcetime": 86400,
+  "triggers": [
+    "https://dcdn.example.com/triggers/0",
+    "https://dcdn.example.com/triggers/1"
+  ]
+}
 ]]></artwork>
    </figure>
    </section>
 
-   <section title="Filtered Collections of Trigger Status Resources" anchor="filtered-collections"><t>
+   <section title="Filtered Collections of Triggers" anchor="filtered-collections"><t>
```

Comment:
I will note that one thing we are lacking is an example of what extended trigger status looks like.  Is it worth providing an example of this?

Reply:
Agree. Need to review Examples for additional gaps as well.